java/code/src/com/redhat/rhn/common/db/datasource/xml/System_queries.xml

```diff
@@ -2338,4 +2338,12 @@ SELECT SP.server_id ID, S.name
   </query>
 </write-mode>
 
+<mode name="count_outdated_systems">
+    <query>
+        SELECT COUNT(DISTINCT(SO.server_id)) AS count
+        FROM rhnServerOverview SO
+        WHERE SO.outdated_packages > 0
+    </query>
+</mode>
+
```

Comment:
It would be nice to avoid adding more queries to the `*_queries.xml` file as these are using a custom-made 
thing replicating Hibernate. At some point in time it would be good to get rid of them all.

Also please use `suseSystemOverview` where possible: the old `rhnServerOverview` view should be phased out too.

Reply:
Could we refactor it together with other queries as a part of the dedicated refactoring task? I think it is better to keep the query here for the sake of consistency as I can see multiple other executions like this one.